src/ray/raylet/scheduling/cluster_task_manager.cc

```diff
@@ -243,7 +243,7 @@ void ClusterTaskManager::DispatchScheduledTasksToWorkers(
                        << worker->WorkerId();
         auto reply = std::get<1>(*work_it);
         auto callback = std::get<2>(*work_it);
-        Dispatch(worker, leased_workers_, allocated_instances, task, reply, callback);
```

Comment:
Compared to fix this typo, I'd more like to remove `leased_worker` parameter in this method as it is fully useless.

Reply:
I agree we should only have one. 

There's a TODO to get rid instance variable in the class.

I don't really care which one we keep, but if we keep the instance variable, can we make it a shared ptr? We should probably avoid saving refs to avoid memory safety issues in the future.